### PR TITLE
logging improvements: replace root logger, fix for aggregation, cleanup

### PR DESCRIPTION
**Pull request by zokalo**
Some logging improvements:
- replaced root logger with named logger. This is required for granular logging setup when this module is included in to an other projects.
- replaced some log messages with "percent formatted" strings with separated set of args. It is crucial for proper logs aggregation. New log-monitoring-system event will be generated for each log message otherwise.
- changed several log messages to include exception body, stack trace
- cleanup a little bit

### CODE REVIEW

File: TikTokApi/browser_utilities/browser.py [hunk 1]
```diff
@@ -80,7 +83,7 @@ def __init__(
                 args=self.args, **self.options
             )
         except Exception as e:
-            logging.critical(e)
+            log.critical("Webkit launch failed", exc_info=True)
```
Comment (zokalo) [inline TikTokApi/browser_utilities/browser.py:86]: added exc stack trace & body with `exc_info=True`

File: TikTokApi/browser_utilities/browser.py [hunk 2]
```diff
@@ -248,7 +251,7 @@ def clean_up(self):
         try:
             self.browser.close()
         except Exception:
-            logging.info("cleanup failed")
+            log.exception("cleanup failed")
```
Comment (zokalo) [inline TikTokApi/browser_utilities/browser.py:254]: added exc stack trace & body with `log.exception` which includes `exc_info=True` to provide more debugging info

File: TikTokApi/utilities.py [hunk 3]
```diff
@@ -2,6 +2,8 @@
 import sys
 import pkg_resources
 
+LOGGER_NAME = 'TikTokApi'
```
Comment (zokalo) [inline TikTokApi/utilities.py:5]: defined common logger name to replace root logger

**Comment by davidteather** [comment]
Implemented these changes with [commit](https://github.com/davidteather/TikTok-Api/commit/5bb0a642fad6749d5b4d0d2ad7a1b584997a75f6) on v5.0.0 branch. 

Added you as a contributor on the commit itself instead of trying to resolve merge conflicts between master/v5.0.0 since v5.0.0 has a lot of project restructuring.

**Pull request closed without merging**